SmartDeviceLink/SDLHapticInterface.h

```diff
@@ -0,0 +1,17 @@
+//
+//  SDLHapticInterface.h
+//  SmartDeviceLink-iOS
+//
+//  Copyright © 2017 smartdevicelink. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "SDLManager.h"
+
+@protocol SDLHapticInterface <NSObject>
+
+- (instancetype)initWithWindow:(UIWindow *)window  sdlManager:(SDLManager *)sdlManager;
```

Comment:
I'm not 100% sure if this will take an `SDLManager` but at least the method should be `initWithWindow:connectionManager:` to match the other managers such as `SDLFileManager`.